### ISSUE TITLE
Quell Register

### ISSUE THREAD

**Issue by julsas** [medizininformatik-initiative/kerndatensatzmodul-studie#4]
https://github.com/medizininformatik-initiative/kerndatensatzmodul-studie/blob/d511f5f6ff57547f2ff65ce8905c00e38231d2e6/input/fsh/Library.fsh#L38

Es gibt einen Suchparameter 'Quellregister' für eine 'Quell Register' Extension, die aber in dem Profil nicht angegeben ist.
Vorschlag: die Extension ins Library Profil aufnehmen.

**Comment by beanformer** [comment]
Einfach das hinzufügen?

Extension: MII_EX_Studie_Quell_Register
Id: mii-ex-studie-quell-register
Title: "MII EX Studie Quell Register"
* insert addMetadata
* value[x] only boolean

(Irgendwo war diese Extension schon mal, ich erinnere mich an die Diskussion, was das überhaupt sein soll, das Quellregister.)

**Comment by julsas** [comment]
> Einfach das hinzufügen?
> ```
> Extension: MII_EX_Studie_Quell_Register Id: mii-ex-studie-quell-register Title: "MII EX Studie Quell Register"
> 
>     * insert addMetadata
> 
>     * value[x] only boolean
>  ```
> 
> (Irgendwo war diese Extension schon mal, ich erinnere mich an die Diskussion, was das überhaupt sein soll, das Quellregister.)

Ja, das sieht korrekt aus. Bei Simplifier findet man die Extension auch noch. Deswegen war ich verwirrt, wo diese zu verwenden ist.
Im Profil MII_PR_Studie_Register sollte dann noch sowas eingefügt werden:

```
* extension contains mii-ex-studie-quell-register named quellRegister 0..1
```
